<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCreateRuleDialog</class>
 <widget class="QDialog" name="PCreateRuleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创建或修改一个规则模版</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="3">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>规则备注</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="editCommect"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPlainTextEdit" name="editScript">
       <property name="styleSheet">
        <string notr="true">
border-color: rgb(0, 255, 127);</string>
       </property>
       <property name="plainText">
        <string>/*请输入sql脚本*/
SELECT * FROM INFORMATION_SCHEMA.VIEWS;
</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>规则模版名称</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editName">
       <property name="text">
        <string>testCreateRule</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>382</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnRun">
         <property name="text">
          <string>运行</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>适用于</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combDBType"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>连接名</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combConnectName"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;表明规则测试连接以外的连接是否适用这个规则&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>是否适用于其他连接</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">QGroupBox { border: none; }</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="radioTrue">
            <property name="text">
             <string>是</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3" columnstretch="6,1,0,1,0,1">
     <item row="0" column="0" colspan="6">
      <widget class="QTabWidget" name="tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tabResult">
        <attribute name="title">
         <string>执行结果预览</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tabChart">
        <attribute name="title">
         <string>图表视图预览</string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelStatus">
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>205</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>106</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>105</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actRun">
   <property name="text">
    <string>执行脚本</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>PCreateRuleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>777</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
